src/utils/constants/formSteps.ts

```diff
@@ -8,8 +8,8 @@ export const formSteps = [
       'legalStatus',
       'agencyInfo',
       'directorName',
-      'open',
-      'close',
+      'hours.open',
```

Comment:
I think that you should be able to consolidate these fields as just 'hours' so that we can perform validation on the whole hours zod object if we ever need to